modules/mono/editor/GodotTools/GodotTools/Build/BuildManager.cs

```diff
@@ -204,6 +205,23 @@ private static bool Publish(BuildInfo buildInfo)
             }
         }
 
+        private static string GetRuntimeConfig()
+        {
+            try
+            {
+                var runtimeConfigPath = Path.Join(GodotSharpDirs.ResTempAssembliesDir,
+                    GodotSharpDirs.ProjectAssemblyName + ".runtimeconfig.json");
+                using var file = Godot.FileAccess.Open(runtimeConfigPath, Godot.FileAccess.ModeFlags.Read);
+                if (file == null)
+                    return null;
+                return file.GetAsText();
+            }
+            catch (IOException)
```

Comment:
I don't think `Godot.FileAccess` throws exceptions. Did you mean to use `System.IO`?